pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[project]
+name = "cdm-spark-standalone"
+version = "0.1.0"
+description = "CDM Spark Standalone"
+readme = "README.md"
+requires-python = ">=3.11"
```

Comment:
Now I think about it we may want to lock this to 3.11 since that's what's in the bitnami containers

Reply:
👍